diff --git a/index.ts b/index.ts
index e502738bb0929a42b4c44241b495d79b97d90eaf..b163928ac090a4e38c18ac03b7734e912d2a009c 100644
--- a/index.ts
+++ b/index.ts	
@@ -50,7 +50,8 @@ const remarkMdxImages: Plugin<[RemarkMdxImagesOptions?], Root> =
               type: 'ImportDefaultSpecifier',
               local: { type: 'Identifier', name }
             }
-          ]
+          ],
+          attributes: []
         })
         imported.set(url, name)
       }
